src/platforms/posix/drivers/df_lsm9ds1_wrapper/df_lsm9ds1_wrapper.cpp

```diff
@@ -572,26 +570,22 @@ int DfLsm9ds1Wrapper::_publish(struct imu_sensor_data &data)
 
 	math::Vector<3> vec_integrated_unused;
 	uint64_t integral_dt_unused;
-
+	// apply sensor rotation on the accel measurement
+	rotate_3f(_rotation, data.accel_m_s2_x, data.accel_m_s2_y, data.accel_m_s2_z);
```

Comment:
I have tested the `rotate_3f` with Navio2 on RPi3. It seems fine. 
Actually, the problem might induce the sensors calibration error is the order between rotation and calibration.  The rotation should be done before calibration, isn't it?
And there is a same problem in `df_mpu9250_wrapper`. If you agree with that, I would like to make a PR for both of them with `rotation_matrix`or `rotate_3f`.